MATURITY.md

```diff
@@ -83,18 +83,18 @@ Mentors and community members are encouraged to contribute to this page and comm
+| ID   | Description                                                                                                                                                                                                               | Status                                                                                                                                                                                                                      |
+|------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| CS10 | The project maintains a public list of its contributors who have decision power. The project's PMC (Project Management Committee) consists of those contributors.                                                         | :white_check_mark: Committers and PPMC members are listed on the website’s community page, see [newcommitter](https://kyuubi.apache.org/become_committer.html), [newppmc](https://kyuubi.apache.org/become_pmc_member.html) |
+| CS20 | Decisions require a consensus among PMC members and are documented on the project's main communications channel. The PMC takes community opinions into account, but the PMC has the final word.                           | :white_check_mark: The project has been making important decisions on the mailing lists.                                                                                                                                    |
+| CS30 | The project uses documented voting rules to build consensus when discussion is not sufficient.                                                                                                                            | :white_check_mark: The project uses the standard ASF voting rules.                                                                                                                                                          |
+| CS40 | In Apache projects, vetoes are only valid for code commits. The person exercising the veto must justify it with a technical explanation, as per the Apache voting rules defined in CS30.                                  | :white_check_mark: The veto is used for code commits only, and each veto with a clear technical explanation.                                                                                                                |
```

Comment:
Let's change this sentence from passive voice to active.

We've used vetoes with clear technical explanations for code commits only.

Reply:
Thanks, updated